### PR TITLE
Upload disabled tests condensed and slow tests to s3

### PR DESCRIPTION
**Pull request by clee2000**
upload disabled tests and slow tests jsons to s3

### CODE REVIEW

File: .github/workflows/update_disabled_tests.yml [hunk 1]
```diff
@@ -41,3 +41,9 @@ jobs:
           user_email: "test-infra@pytorch.org"
           user_name: "Pytorch Test Infra"
           commit_message: "Updating condensed disabled tests stats"
+
+      - name: Upload file to s3
+        run: python3 .github/scripts/upload_to_s3.py "disabled-tests-condensed.json"
```
Comment (huydhn) [inline .github/workflows/update_disabled_tests.yml:46]: Hmm, I don't think you need to have a Python script for such simple operation, you could try to install `awscli` with pip and just run `aws s3 cp debug.txt s3://ossci-metrics/debug.txt`

File: .github/workflows/update_disabled_tests.yml [hunk 2]
```diff
@@ -41,3 +41,9 @@ jobs:
...
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```
Comment (huydhn) [inline .github/workflows/update_disabled_tests.yml:48]: Nit: I think if we are using self-hosted runner here, i.e. linux.large, you don't even need to set the key as the runner has read/write access to S3 (I just test it out)

**Review by huydhn — APPROVED** [review]
There are some minor comments, but overall LGTM!

File: .github/workflows/update_disabled_tests.yml (hunk 2 above)
Comment (clee2000) [reply .github/workflows/update_disabled_tests.yml:50]: theres are run on github provided runners

**Pull request merged**